spec/fixtures/pulpcore-migration-dont-proxy-yum/katello-answers-before.yaml

```diff
@@ -32,6 +32,7 @@ foreman_proxy::plugin::salt: false
 foreman_proxy_content:
   proxy_pulp_isos_to_pulpcore: false
   puppet: false
+  enable_deb: true
```

Comment:
My initial reaction was, the minimal change to get the tests to work would be to omit this and add only `proxy_pulp_deb_to_pulpcore: false` to the katello-answers-after.yaml. My thinking was that adding `enable_deb: true` to the katello-answers-before.yaml fixture somewhat overloads the test as now it's also testing the new migration that you created.

However, thinking about it some more, I think that's actually fine, but let's also rename the test if we go that route. It should be sufficient to rename the test in `spec/migration_spec.rb` on L87 to `"pulpcore migration dont use content plugins on upgrades"` which is a better name I think.

Then also, `# git mv spec/fixtures/pulpcore-migration-{dont-proxy-yum,dont-proxy-content-on-upgrades}` and update the paths on L88 and L91 in `spec/migration_spec.rb` accordingly.

Reply:
updated!